src/components/AppNavigation/AppointmentConfigList/AppointmentConfigListItem.vue

```diff
@@ -29,6 +29,11 @@
 				<CalendarCheckIcon :size="20" decorative />
 			</template>
 			<template #actions>
+				<ActionLink :href="config.bookingUrl"
+					icon="icon-external"
```

Comment:
Please use a material design icon here. `OpenInNew` would be an option here:
https://materialdesignicons.com/icon/open-in-new